packages/cypress/package.json

```diff
@@ -43,14 +44,17 @@
     "typescript": "^5.5.2"
   },
   "dependencies": {
-    "@replayio/replay": "workspace:^",
     "@replayio/test-utils": "workspace:^",
     "chalk": "^4.1.2",
     "debug": "^4.3.4",
+    "fs-extra": "^11.2.0",
```

Comment:
It's not a new thing here but it would be great to get rid of this dependency. I feel like it adds too much to install size for what we are actually using it for

Reply:
Quick audit of the APIs we use from this package:
* `appendFileSync`
* `createReadStream`
* `ensureDirSync`
* `ensureFileSync`
* `existsSync`
* `readdir`
* `readdirSync`
* `readFile`
* `readFileSync`
* `ReadStream`
* `removeSync`
* `renameSync`
* `rmSync`
* `stat`
* `unlinkSync`
* `writeFileSync`

I don't think removing it is worth spending time on. Saving `8kB` (minified) is nothing for developer' machines or CI boxes.